components/date_input/README.md

```diff
@@ -0,0 +1,161 @@
+---
+description: Date input allows users to enter an already known date
+title: Date Input
+status: Draft
+---
+
+# Date Input
+
+Date input fields allow users to enter an already known date
+
+### Required
+
+## Last Updated:
+
+## Visual
+Date Input Visual:
+![Date Input](https://github.com/bcgov/design-system/blob/master/components/date_input/images/date_input.png?raw=true)
+
+## Requirements
+* This component must be used according the the standard outined in this document. Any feedback should be directed to the feedback forum.
+
+## How to Use This
+*	Use when users need to enter an easily memorable date
+
+## Don't Use This For
+*	Dates that cannot be easily remembered
+*	When users have to schedule something in the future, a calendar dropdown may be better.
```

Comment:
I wonder if we should recommend a calendar dropdown here, as it isn't something we've tested or researched. Could we just say "another option may be better", and wait until we have a more specific recommendation. I'm concerned that a calendar dropdown may not actually be the right or accessible choice